src/main/java/net/phptravels/gui/pages/DashboardPage.java

```diff
@@ -1,4 +1,4 @@
-package net.phptravels.gui.login;
+package net.phptravels.gui.pages;
 
```

Comment:
we should not define driver in page objects
remove on line 10 `protected WebDriver driver;`